internal/sidekick/internal/rust/annotate.go

```diff
@@ -437,6 +441,9 @@ type fieldAnnotations struct {
 	// If true, this is a `wkt::NullValue` field, and also requires super-extra
 	// custom deserialization.
 	IsWktNullValue bool
+	// If this field is part of a oneof group, this will contain the other fields
+	// in the group.
+	OtherFieldsInGroup []*api.Field
```

Comment:
I've been thinking maybe we can move this (and a couple others I've added for samples) as property or functions in the model. These are not language specific and could benefit any language that wants to generate samples.
What do folks things about that?

I'm happy to send a follow up PR with that later.